cmd/kube-apiserver/apiserver.go

```diff
@@ -29,6 +31,17 @@ import (
 )
 
 func main() {
+	if godebug := os.Getenv("GODEBUG"); !strings.Contains(godebug, "x509sha1") {
+		// match go1.17 default behavior on release-1.23
+		// see https://go.dev/doc/go1.18#sha1
+		os.Setenv("GODEBUG", strings.TrimPrefix(godebug+",x509sha1=1", ","))
```

Comment:
does this work when set after the runtime is already initialized?

Reply:
yes, in go1.19.4+ :)

xref https://go-review.git.corp.google.com/c/go/+/445656

Reply:
https://github.com/golang/go/commit/db5cb5fceed48a487ca15c8321ddcb19652cc1b9 for public git

Excellent 🎉

Reply:
gah... sorry... updated the link